source/01-LE/03.ptx

```diff
@@ -9,6 +9,30 @@
         </ul>
     </objectives>
 
+<subsection>
+    <title>Warm Up</title>
+    <activity>
+        <task>
+            <statement>
+                <p>
+                    Without referring to your Activity Book, which of the four criteria for a matrix to be in Reduced Row Echelon Form (RREF) can you recall?
```

Comment:
Two things this brings up for me:

- I usually prefer activities that are more flexible in what's "allowed" so instructors can use them as appropriate in their unique instructional contexts. I've had teams where I'd be grateful if they would look up the answer rather than just sit still and not engage.
- I haven't been giving feedback on "4S" structure, as I haven't considered these warmups/cooldowns as TBIL activities. However, we will want to make that distinction clear at some point before the fall so instructors know how to use them appropriately.

Reply:
While I take your point for the first bullet, I'd say that the second task addresses this point. Teams that can't recall any can "move on" to the next activity in which it says explicitly to look up the the criteria that they can't recall. 

For the second point, I agree that that we should make the distinction clear. Where should that go? Do we have a "for instructors" section?

Reply:
We need to make an instructor edition; I've got the machinery set up for precalculus, which I should port here.

Reply:
I think this is fine as-is for now.